scripts/benchmark-size/scope.json

```diff
@@ -0,0 +1,51 @@
+[
+  { "package": "@aws-sdk/credential-providers" },
+  { "package": "@aws-sdk/credential-provider-cognito-identity" },
+  { "package": "@aws-sdk/credential-provider-env", "skipBundlerTests": true },
```

Comment:
`skipBundlerTests` should be removed because we now need to generate 2 reports. The same applies to scope. We should have `browser-scope.json` for pckages supports browser and `scope.json` for all publish packages.